docs/source/building_an_experiment.rst

```diff
@@ -2,49 +2,306 @@
+    The default is 'mean'. 
 
-..  TODO determine where users should write their experiments
-    because that will affect the import statement - putting them
-    directly in experiments?
+comment *defaults*
+    a comment string that will be placed in the borealis files describing the slice. Defaults
+    to empty string.
+
+lag_table *defaults*
+    used in acf calculations. It is a list of lags. Example of a lag: [24, 27] from
+    8-pulse normalscan. This defaults to a lagtable built by the pulse sequence 
+    provided. All combinations of pulses will be calculated, with both the first pulses
+    and last pulses used for lag-0.
+
+pulse_phase_offset *defaults*
+    Allows phase shifting between pulses, enabling encoding of pulses. Default all
+    zeros for all pulses in pulse_sequence.
+
+range_sep *defaults*
+    a calculated value from pulse_len. If already set, it will be overwritten to be the correct
+    value determined by the pulse_len. Used for acfs. This is the range gate separation,
+    in azimuthal direction, in km.
+
+rx_int_antennas *defaults*
+    The antennas to receive on in interferometer array, default is all
+    antennas given max number from config.
+
+rx_main_antennas *defaults*
+    The antennas to receive on in main array, default is all antennas
+    given max number from config.
+
+scanbound *defaults*
+    A list of seconds past the minute for integration times in a scan to align to. Defaults
+    to None, not required.
+
+seqoffset *defaults*
+    offset in us that this slice's sequence will begin at, after the start of the sequence.
+    This is intended for PULSE interfacing, when you want multiple slice's pulses in one sequence
+    you can offset one slice's sequence from the other by a certain time value so as to not run both
+    frequencies in the same pulse, etc. Default is 0 offset.
+
+tx_antennas *defaults*
+    The antennas to transmit on, default is all main antennas given max
+    number from config.
+
+xcf *defaults*
+    flag for cross-correlation data. The default is True if acf is True, otherwise False.
+
+
+**Read-only Slice Keys**
+
+clrfrqflag *read-only*
+    A boolean flag to indicate that a clear frequency search will be done. 
+    **Not currently supported.**
+
+cpid *read-only*
+    The ID of the experiment, consistent with existing radar control programs.
+    This is actually an experiment-wide attribute but is stored within the 
+    slice as well. This is provided by the user but not within the slice, 
+    instead when the experiment is initialized.
+
+rx_only *read-only*
+    A boolean flag to indicate that the slice doesn't transmit, only receives.
+
+slice_id *read-only*
+    The ID of this slice object. An experiment can have multiple slices. This 
+    is not set by the user but instead set by the experiment when the 
+    slice is added. It will be unique to an experiment run. When experiments
```

Comment:
"It will be unique to an experiment run"  This sentence is a bit confusing to me. Could you instead put something like: "Each slice id within an experiment is unique" ?